packages/superset-ui-chart/src/@types/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@superset-ui/core';
```

Comment:
what's the reasoning with adding the `@` prefix to the index file? is that best practice/standard? (tried to google for it but didn't see anything, a bit hard to search for). in `connection we have a `types.ts` file without an `@`.

Reply:
This is to manually declare types and fix the error for external module that do not have types. 

```
error TS7016: Could not find a declaration file for module '@superset-ui/core'
```

https://medium.com/@chris_72272/migrating-to-typescript-write-a-declaration-file-for-a-third-party-npm-module-b1f75808ed2

Reply:
Seems like it works if I just move this file to `src` with any name but extension `.d.ts` as well.